app/controllers/api/v3/search.rb

```diff
@@ -98,6 +98,30 @@ def forwarded_ip_address
           close_serials = Bike.search_close_serials(interpreted_params)
           serialized_bikes_results(paginate(close_serials))
         end
+
+        desc "Search external registries", {
+               notes: <<-NOTE,
+                 This endpoint accepts a serial number and searches external registries for it.
+
+                 It returns matches found.
+               NOTE
+             }
+        paginate
+        params do
+          requires :serial, type: String, desc: "Serial, homoglyph matched"
+          optional :per_page, type: Integer, default: 25, desc: "Bikes per page (max 100)"
```

Comment:
I think we should accept all the search parameters, and we can improve how accurately they match as time goes on.

I think we should launch with the proximity parameters.